README.md

```diff
@@ -5,73 +5,41 @@ Licensed under the Apache License, Version 2.0 (the LICENSE-APACHE file) or
-   a complementary set of use-cases when compared to `slab-alloc`.
-
-* `bsalloc` is a very simple dynamic memory allocator used to bootstrap
-  `elfmalloc` in the cases when it depends on libraries that allocate memory on
-  the heap. Using `bsalloc` allows our code to be completely self-hosting (not
-  relying on whatever `malloc` Rust would otherwise use), without having to
-  re-write key library infrastructure like `crossbeam` that relies on a dynamic
-  memory allocation.
-
-* `mmap-alloc` is an allocator that constructs anonymous virtual memory maps.
-  There are enough edge-cases and cross-platform issues with calling `mmap`
-  directly, that it made sense to make this its own crate.
-
-## Utility Crates
-
-* `bagpipe` provides a number of key concurrent queue-like data-structures. In
-  addition to implementing a number of state-of-the-art concurrent MPMC queues
-  in the `crossbeam` framework, it also exposes a new data-structure: the
-  `Bagpipe`. `Bagpipes` are like queues, but they trade consistency for
-  scalability. This is normally a dubious exchange to make, but for
-  `elfmalloc`, `Bagpipe`s are used as "free lists" where ordering on the
-  insertion and removal of elements is not required.
-
-* `object-alloc` includes a number of traits currently used to define
-  object-specific allocators. Its traits are currently only implemented by the
-  allocators in `slab-alloc`, though we intend this to change over time.
-
-* `object-alloc-test` provides a number of tests that check for correctness of
-  an arbitrary object allocator (i.e. an allocator implementing one of the
-  traits in `object-alloc`).
-
-* `malloc-bind` is a crate that provides macros to transform any general
-  allocator that can store its own size and alignment-related metadata into a
-  full `malloc` implementation. The intent is to replace `elfc` with this more
-  generic solution in the future.
+# Allocators in Rust
+
+[![Build Status](https://travis-ci.org/ezrosent/allocators-rs.svg?branch=sysconf)](https://travis-ci.org/ezrosent/allocators-rs)
+[![Build status](https://ci.appveyor.com/api/projects/status/github/ezrosent/allocators-rs?svg=true)](https://ci.appveyor.com/project/ezrosent/allocators-rs)
+
+_Looking for elfmalloc in particular? It's [here](https://github.com/ezrosent/allocators-rs/blob/master/elfmalloc)._
+
+This repository encompasses a number of different crates. Some are general
+memory allocators, some are object allocators (allocators which allocate a
+specific type of object), and some are utility crates providing various
+features needed to implement these allocators.
+
+The [`info`](https://github.com/ezrosent/allocators-rs/blob/master/info) directory contains more detailed information, including performance
+measurements. Aside from `info`, all top-level directories are Rust crates.
+More detailed information on each crate can be found in the crate's `README.md`.
```

Comment:
`elfc` should be mentioned somewhere

Reply:
Also, `elfc`'s README should be updated now that it is a thin wrapper around `malloc-bind`